docs/openapi/components/schemas/common/CBPEntrySummary.yml

```diff
@@ -199,10 +199,10 @@ properties:
   importerNumber:
     title: Importer Identifier
     description: Record the IRS EIN, SSN, or CBP assigned number of the importer of record. Proper format is listed under the instructions for Consignee Number.
-    type: string
+    $ref: ./CBPImporterOfRecord.yml
```

Comment:
Why would we do this? 
This is a CBP entry summary when would you not have the importerNumber be the CBPImporter of Record?

Reply:
Indeed you have one of the applicable importer identifiers: IRS EIN, SSN, or CBP assigned. The class is to allow indication which one it is. We use it elsewhere, this is to make 7501 consistent.